### ISSUE TITLE
Form prefill prevents message submission (v1.0.2)

### ISSUE THREAD

**Issue by lukevear** [helpscout/beacon-ios-sdk#47]
* Beacon install method: [X] Manual

**Context**
- v1.0.2
- Secure mode is enabled

**Actual outcome**
Beacon does not allow the user to hit "Send A Message" if the pre-fill feature is used. After setting `form.subject` & `form.text` the beacon is displayed, but "Send A Message" is greyed out and tapping it shows a generic "Please complete all fields" error message (see attached image).

Additionally, if `form.message` is NOT pre-filled, submission still fails with the same error. This can be "worked-around" with the following snippet:

```
-(void)prefill:(HSBeaconContactForm *)form {
    form.subject = self.subject;
    // form.text = self.text;

    if ([HSBeacon currentUser] != nil) {
    	form.email = [HSBeacon currentUser].email;
    	form.name = [HSBeacon currentUser].name;
    }
}
```

**Expected outcome**
Beacon should only show the generic "Please complete all fields" error message if the user removes the pre-filled message.

**Screenshots or GIFs**
![UNADJUSTEDNONRAW_thumb_8f3](https://user-images.githubusercontent.com/11508219/69511540-7e257700-0f7b-11ea-87fa-da62e17c6124.jpg)

**Comment by lukevear** [comment]
I should also add that we call `[HSBeacon reset]` after the form is closed to allow subsequent pre-filling based on where the user is within our app, however the issue remains regardless of whether or not `[HSBeacon reset]` has been called previously.

**Comment by hs-gv** [comment]
Hello, thank you for taking the time to report this.  We will attempt to reproduce this issue and get back to you with an update.

**Comment by lukevear** [comment]
Thanks! Let me know if you need any more information from my end.

**Comment by samsymons** [comment]
@lukevear Thanks again for the report here – we should have this fixed up for an upcoming Beacon release soon!

Internal Jira reference: https://helpscout.atlassian.net/browse/IOSSDK-49

**Comment by lukevear** [comment]
Thanks @samsymons - looking forward to it.

**Comment by JonasWho** [comment]
@samsymons Do you have an updated ETA on the next release with a fix for this?

**Comment by codenameDuchess** [comment]
@JonasWho the ticket for this fix is part of our current dev cycle which means ETA is likely to be early January (or sooner if we can get it in before our holiday freeze). 

Will update this issue with the fix version and a more concrete ETA once we have one.

**Comment by bvirlet** [comment]
Do you have any update on that? Support request have been broken in our app for more than a month.

**Comment by shaunmcilroy** [comment]
Hi @bvirlet - just responded to your email. We hope to have a fix for this issue out in our next release coming soon. I don't have an ETA just yet on day/time but we'll be in touch with you and the rest of the folks on this thread when it's live. Thanks for your patience! 👍

**Comment by samsymons** [comment]
This issue should now be fixed in the latest Beacon version, 1.0.3. Thanks for the patience here, and please let us know if you run into any other issues!
